meson.build

```diff
@@ -93,9 +93,13 @@
 #
 # Then to install:
 #    meson install
-# Or, on Linux, you can do:
-#    sudo meson install
-# which avoids the pop-up window
+#
+# Note that, on Linux, although you can do `sudo meson install` to avoids the pop-up window prompting you for a
+# password to allow meson to do sudo itself, this is not a good idea.  Aside from the general principle of not
+# running-as-root for any longer than we need to, the problem is that some files in the mbuild tree (in particular
+# meson-logs/install-log.txt) will be created as, and thus owned by, root, with no write access for other users.  If you
+# then run meson again without sudo, you'll then get an error if it tries to write to file.  This certainly happens if
+# you run `bt package` for example.
```

Comment:
The install-log.txt sort of has to be owned by root, because it records files installed by a root process. No other files should be owned by root, but before the upcoming meson 1.1.0 if you ran `sudo meson install` it would re-run the build stage as root, and possibly mess a lot of things up.

(Meson 1.1.0 will also bring support for using sudo instead of a pop-up window -- that pop-up window is actually polkit, not sudo.)

Reply:
Ah, OK, thanks -- good to know.